bundle/manifests/kmm-operator-build-and-sign_networking.k8s.io_v1_networkpolicy.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: build-and-sign
```

Comment:
We already shipped NP for operands, we should now only add NP manifests to the bundle for the operator itself (controllers + webhooks)